libs/ConfigKit/include/ConfigComplex.h

```diff
@@ -0,0 +1,34 @@
+// Leka - LekaOS
+// Copyright 2023 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <filesystem>
+#include <span>
+
+namespace leka {
+
+template <std::size_t SIZE>
+struct ConfigComplex {
```

Comment:
I'm wondering about the necessity to add yet another struct which would make Config vs ConfigComplex difficult to grasp.

In reality `Config == ConfigComplex<1>`

I would prefer we make `Config` evolve to be of arbitrary size and refactor it's current use with `Config<1>` rather than adding two different types of Config.